Animationscroll

UlyanaShnyr opened the pull request.

Empty-Fox committed:
modifity

Empty-Fox committed:
animation burger and matrix

UlyanaShnyr committed:
Update style.css

Empty-Fox committed:
simple work with view

Empty-Fox committed:
simple

Empty-Fox committed:
simple

Empty-Fox committed:
simple

Empty-Fox committed:
added few animation to scroll and created animetion SVG

ValeriiKupyna commented on css/style.css:
```diff
@@ -1,22 +1,24 @@
     border-radius: 50%;
     background-color:black;
-    transition: 0.9s
+    transition: 0.9s;    
```
видали лишні пробіли

ValeriiKupyna commented on css/style.css:
```diff
@@ -1,22 +1,24 @@
     margin: 0 auto;
     
 }
+body{   
```
Відформатуй код: (Пробіл після назви класу, тегу ... Відступ до і після)

.classname {
...
}

.classname {
...
}

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
-    transition: 0.9s;
+    overflow-y: auto;  
+    display: none;      
+    
```
видали лишній відступ

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
+    
 }
+
+
```
видали лишній відступ

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
 .burger-menu-active .burger-menu-nav{
-left:0;
+    display: block;
+    width:50%;
```
пропучений відступ.

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
+    100% {
+        transform: translateX(0%);             
+    }       
+    
```
видали лишній відступ

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
 .burger-menu-link{
     padding: 30px;
-    font-family: 'Courier New', Courier, monospace;
+    font-family:inconsolata,monospace;
```
пропущені відступи

ValeriiKupyna commented on css/style.css:
```diff
@@ -57,33 +58,54 @@
     color: white;
-
+    display: flex;
+     margin-left: 10%; 
```
вирівняй будь ласка

ValeriiKupyna commented on css/style.css:
```diff
@@ -112,9 +135,63 @@ section{
     color: black;
-   
+    overflow:hidden;
+    transition: all 3s;
```
Старайся прописувати для яких саме проперті тобі потрібен transition

ValeriiKupyna commented on css/style.css:
```diff
@@ -112,9 +135,63 @@ section{
 }
 
-  footer{
+#home{
```
Старайся використовувати класи.

ValeriiKupyna commented on index.html:
```diff
@@ -23,13 +27,62 @@
+        </section>
+        <section id="projects" class="section" ><p class="animated mov">PROJECTS</p></section>
+        <section id="about" class="section" class="animated mov">
+                <div id="svg" style="width: 40%; margin-left: 10%; margin-top: 10%" class="animated mov">    
```
Відформатуй код будь ласка. + використовувати інлайн стилі в даму випатку не є саме хороше рішення.

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+            $(this).addClass('rotateIn');
+        }
+        
+        console.log(this)
```
Видали лишній пробіл і консоль лог

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
```
Оскільки дані не модифікуються в подальшому використовуй const

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
```
добав відступи до і після операторів

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+            $(this).addClass('fadeInRight');
+        }
+        
+        console.log(this)
```
Видали лишній пробіл і консоль лог

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+    $('.mov1').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
```
Видали лишній пробіл і консоль лог

ValeriiKupyna commented on js/animation_scroll.js:
```diff
@@ -0,0 +1,22 @@
+
+    $('.mov1').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
```
добав відступи до і після операторів

ValeriiKupyna commented on js/burgermenu.js:
```diff
@@ -9,7 +6,8 @@ let linkAbout=document.getElementById('nav-about');
 button.onclick = function() {
-    menu.classList.toggle('burger-menu-active');   
+    menu.classList.toggle('burger-menu-active'); 
+   
```
добав відступи до і після операторів, для змін які не будеш переприсвоювати використовюй const.
видали лишній відступ.

ValeriiKupyna commented on js/burgermenu.js:
```diff
@@ -19,7 +17,8 @@ linkProject.onclick = function() {
 linkAbout.onclick = function() {
-    menu.classList.toggle('burger-menu-active');   
+    menu.classList.toggle('burger-menu-active'); 
+      
```
видали лишній відступ.

ValeriiKupyna commented on js/matrix.js:
```diff
@@ -0,0 +1,34 @@
+let canvas_element = document.getElementById('canvas_element');
```
читай зауваження вище.

ValeriiKupyna commented on js/matrix.js:
```diff
@@ -0,0 +1,34 @@
+
+let font = 'arial';
+let fontSize = 15;
+matrix_context.font = fontSize + 'em' + font;
```
matrix_context.font = `${fontSize}em${font}`

ValeriiKupyna commented on js/matrix.js:
```diff
@@ -0,0 +1,34 @@
+let symbols;
+symbols = '0123456789ABCDEF'; 
+symbols = symbols.split(''); 
+
```
const symbols = '0123456789ABCDEF'.split(''); хоча чому просто незаписати одразу масив ?

ValeriiKupyna commented on js/matrix.js:
```diff
@@ -0,0 +1,34 @@
+matrix_context.fillStyle = 'rgba(231, 229, 228, 0.05)';
+matrix_context.fillRect(0, 0, canvas_element.width, canvas_element.height);        
+for (let i = 0; i < drops.length; i++) {        
+let char = symbols[Math.floor(Math.random() * symbols.length)];        
```
let -> const

UlyanaShnyr commented:
Наче виправила...але він ще не закінчений...буду дописувати)

06 мая 2019, 19:31:20, от Valerii <notifications@github.com>:

@ValeriiKupyna commented on this pull request.

In css/style.css:

>      z-index:30;
     width:80px;
     height: 80px;
     border-radius: 50%;
     background-color:black;
-    transition: 0.9s
+    transition: 0.9s;    
видали лишні пробіли

In css/style.css:

>  *{
     /* max-width: 1200px;  */
     margin: 0 auto;
     
 }
+body{   
Відформатуй код: (Пробіл після назви класу, тегу ... Відступ до і після)

.classname {
...
}

.classname {
...
}

In css/style.css:

>      height: 100%;
     background-color: black;
-    overflow-y: auto;
-    left: -100%;
-    transition: 0.9s;
+    overflow-y: auto;  
+    display: none;      
+    
видали лишній відступ

In css/style.css:

>  }
+
+
видали лишній відступ

In css/style.css:

>  .burger-menu-active .burger-menu-nav{
-left:0;
+    display: block;
+    width:50%;
пропучений відступ.

In css/style.css:

>  
+@keyframes burger-menu-nav {
+    0% {
+        transform: translateX(100%);        
+    }
+     
+    100% {
+        transform: translateX(0%);             
+    }       
+    
видали лишній відступ

In css/style.css:

>  .burger-menu-link{
     padding: 30px;
-    font-family: 'Courier New', Courier, monospace;
+    font-family:inconsolata,monospace;
пропущені відступи

In css/style.css:

>      font-size: 32px;
     text-decoration: none;
     text-transform: uppercase;
     letter-spacing: 5px;
     font-weight: bold;
     color: white;
-
+    display: flex;
+     margin-left: 10%; 
вирівняй будь ласка

In css/style.css:

> @@ -112,9 +135,63 @@ section{
     font-size: 72px;
     text-transform: uppercase;
     color: black;
-   
+    overflow:hidden;
+    transition: all 3s;
Старайся прописувати для яких саме проперті тобі потрібен transition

In css/style.css:

>  }
 
-  footer{
+#home{
Старайся використовувати класи.

In index.html:

> @@ -23,13 +27,62 @@
             <div id="burger-menu-overlay" class="burger-menu-overlay"></div>
         </div>
     </header>
-    <div class="container">
-        <section id="home" class="section">HOME</section>
-        <section id="projects" class="section">PROJECTS</section>
-        <section id="about" class="section">ABOUT</section>
-        <section id="contact" class="section">CONTACT</section>
+    <canvas style="width:100%; height:100%" id='canvas_element'> </canvas>
+    <div class="container">           
+        <section id="home" class="section">
+             <h5 ><span > Junior</span> Front-End <br> Developer <br><span class="autorName">Uliana Shnyr</span> </h5>  
+        </section>
+        <section id="projects" class="section" ><p class="animated mov">PROJECTS</p></section>
+        <section id="about" class="section" class="animated mov">
+                <div id="svg" style="width: 40%; margin-left: 10%; margin-top: 10%" class="animated mov">    
Відформатуй код будь ласка. + використовувати інлайн стилі в даму випатку не є саме хороше рішення.

In js/animation_scroll.js:

> @@ -0,0 +1,22 @@
+
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
+            $(this).addClass('rotateIn');
+        }
+        
+        console.log(this)
Видали лишній пробіл і консоль лог

In js/animation_scroll.js:

> @@ -0,0 +1,22 @@
+
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
Оскільки дані не модифікуються в подальшому використовуй const

In js/animation_scroll.js:

> @@ -0,0 +1,22 @@
+
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
добав відступи до і після операторів

In js/animation_scroll.js:

> +        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
+            $(this).addClass('rotateIn');
+        }
+        
+        console.log(this)
+    })
+
+    $('.mov1').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
+            $(this).addClass('fadeInRight');
+        }
+        
+        console.log(this)
Видали лишній пробіл і консоль лог

In js/animation_scroll.js:

> +
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
+            $(this).addClass('rotateIn');
+        }
+        
+        console.log(this)
+    })
+
+    $('.mov1').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
Видали лишній пробіл і консоль лог

In js/animation_scroll.js:

> @@ -0,0 +1,22 @@
+
+$(window).scroll(function(){
+    $('.mov').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
+        if(imagePos<topOfWindow+window.innerHeight){
+            $(this).addClass('rotateIn');
+        }
+        
+        console.log(this)
+    })
+
+    $('.mov1').each(function(){
+        let imagePos=$(this).offset().top;
+        let topOfWindow=$(window).scrollTop();
добав відступи до і після операторів

In js/burgermenu.js:

> @@ -9,7 +6,8 @@ let linkAbout=document.getElementById('nav-about');
 let linkContact=document.getElementById('nav-contact');
 let overlay=document.getElementById('burger-menu-overlay');
 button.onclick = function() {
-    menu.classList.toggle('burger-menu-active');   
+    menu.classList.toggle('burger-menu-active'); 
+   
добав відступи до і після операторів, для змін які не будеш переприсвоювати використовюй const.
видали лишній відступ.

In js/burgermenu.js:

> @@ -19,7 +17,8 @@ linkProject.onclick = function() {
     menu.classList.toggle('burger-menu-active');   
 };
 linkAbout.onclick = function() {
-    menu.classList.toggle('burger-menu-active');   
+    menu.classList.toggle('burger-menu-active'); 
+      
видали лишній відступ.

In js/matrix.js:

> @@ -0,0 +1,34 @@
+let canvas_element = document.getElementById('canvas_element');
читай зауваження вище.

In js/matrix.js:

> @@ -0,0 +1,34 @@
+let canvas_element = document.getElementById('canvas_element');
+canvas_element.width=window.innerWidth;
+canvas_element.height=window.innerHeight;
+
+let matrix_context = canvas_element.getContext('2d');
+
+let font = 'arial';
+let fontSize = 15;
+matrix_context.font = fontSize + 'em' + font;
matrix_context.font = ${fontSize}em${font}

In js/matrix.js:

> @@ -0,0 +1,34 @@
+let canvas_element = document.getElementById('canvas_element');
+canvas_element.width=window.innerWidth;
+canvas_element.height=window.innerHeight;
+
+let matrix_context = canvas_element.getContext('2d');
+
+let font = 'arial';
+let fontSize = 15;
+matrix_context.font = fontSize + 'em' + font;
+let cols = canvas_element.width / fontSize;
+
+let symbols;
+symbols = '0123456789ABCDEF'; 
+symbols = symbols.split(''); 
+
const symbols = '0123456789ABCDEF'.split(''); хоча чому просто незаписати одразу масив ?

In js/matrix.js:

> +symbols = '0123456789ABCDEF'; 
+symbols = symbols.split(''); 
+
+let drops = [];
+for (let i = 0; i < cols; i++) {
+    drops[i] = Math.floor(Math.random() * canvas_element.height);
+}
+
+setInterval(rain, 250); 
+
+function rain() {    
+   
+matrix_context.fillStyle = 'rgba(231, 229, 228, 0.05)';
+matrix_context.fillRect(0, 0, canvas_element.width, canvas_element.height);        
+for (let i = 0; i < drops.length; i++) {        
+let char = symbols[Math.floor(Math.random() * symbols.length)];        
let -> const

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
